src/test/java/de/qabel/desktop/ui/remotefs/RemoteFSControllerTest.java

```diff
@@ -66,9 +70,11 @@ public void setUp() throws Exception {
     @After
     public void after() throws Exception {
         localWrite.delete(localStorageFile.getName());
-        File dir = new File("tmp/testFolder");
-        FileUtils.deleteDirectory(dir);
-
+        FileUtils.deleteDirectory(new File(TMP_DIR + "/" + TEST_FOLDER));
```

Comment:
hint:
```JAVA
new File(TMP_DIR + "/" + TEST_FOLDER))
```
is the same as
```JAVA
File tmpDir = new File(TMP_DIR);
new File(tmpDir, TEST_FOLDER);
```
especially useful if tmpDir is a class property (reused). This is semantically preferable over string concatenations.